devtools/platform-descriptor-json/src/main/resources/templates/README.maven.ftl

```diff
@@ -0,0 +1,29 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
+
+## Running the application in dev mode
+
+You can run your application in dev mode that will allow to have hot reload feature using:
+```
+./mvnw compile quarkus:dev
+```
+## Packaging and running the application
+
+The application is packageable using `./mvnw package`. It produces 2 jar files in `/target`:
+- `${project_artifactId}-${project_version}.jar` - containing just the classes and resources of the projects, it’s the regular artifact produced by the Maven build;
```

Comment:
I would go with what you did for the Gradle version and only talk about the -runner-jar.